build.sh

```diff
@@ -33,7 +34,10 @@ lessram=
 arch=$(${thisDir}/get_arch.sh)
 features=
 disablefeatures=
-tests=1
+commitid="${commitid:-local}"
+dpkgArch="${arch:-$(dpkg --print-architecture | awk -F- "{ print \$NF }")}"
+skip_tests=1
```

Comment:
We should probably invert this value, because this way it seems that 1 means that tests are skipped and 0 means they are not.

The current implementation however, uses this variable the other way around.

Reply:
Not sure if that is a good, skip_tests is set to off by default, so it it should be false. In shell/bash true is represented by 0 and false (error) by a none 0 value.
```
true ; echo $?
0
false ; echo $?
1
```